frontend/pages/events/[id]/index.vue

```diff
@@ -127,13 +127,21 @@ const eventButtons: MenuSelector[] = [
   // },
 ];
 
-onMounted(() => {
-  redirectBasedOnScreenSize();
-});
 
-function redirectBasedOnScreenSize() {
+const handleResize = () => {
   if (window.innerWidth > 640) {
+    window.removeEventListener('resize', handleResize)
     navigateTo(`${id}/about`);
   }
 }
+
+onMounted(() => {
+  // Add event listener to handle resizing
```

Comment:
Minor "nit" (nitpick) that there should be a period at the end here for full comments. I can add these myself though, @felipolis! We tend to do full sentences including punctuation for comments that are their own line and lower case the first letter/no punctuation for inline comments (Python styleguide, but it seems to be gaining traction elsewhere).